src/App.js

```diff
@@ -21,15 +22,30 @@ const useStyles = makeStyles((theme) => ({
 export default function App() {
   const classes = useStyles();
 
+  const [shouldRunSUSHI, setShouldRunSUSHI] = useState(false);
+  const [text, setText] = useState('Edit FSH Here!');
+
+  function updateShouldRunSUSHI(runVariable) {
+    setShouldRunSUSHI(runVariable);
+  }
+  function updateTextValue(text) {
+    setText(text);
+  }
+
   return (
     <div className="root">
       <TopBar />
+      <RunButton onClick={updateShouldRunSUSHI} />
       <Grid className={classes.container} container>
         <Grid className={classes.itemTop} item xs={6}>
-          <CodeMirrorComponent />
+          <CodeMirrorComponent
+            value={text}
+            updateTextValue={updateTextValue}
+            updateShouldRunSUSHI={updateShouldRunSUSHI}
+          />
         </Grid>
         <Grid className={classes.itemTop} item xs={6}>
-          <JSONOutput />
+          <JSONOutput value={shouldRunSUSHI} text={text} />
```

Comment:
I should have maybe thought of this before, but I just realized that the prop `value` here is referring to the `shouldRunSUSHI` boolean, but the `value` prop on the CodeMirrorComponent is the text value. I think that difference might be slightly confusing. Since this boolean prop will be a boolean to say SUSHI just ran maybe the prop could be `shouldDisplaySUSHI` or something like it? What do you think?